### DISCUSSION TITLE
Zephyr HCI USB firmware with TX power vendor commands

### DISCUSSION THREAD

**Discussion by mogenson** [google/bumble#274] [Show and tell]
I'm uploading a firmware binary here, because I think I can link to it from a new mkdocs page from this PR: https://github.com/google/bumble/pull/258

[hci_usb.zip](https://github.com/google/bumble/files/12562079/hci_usb.zip)

**Comment by barbibulle** [comment]
The current setup is that the contents of the GitHub pages are loaded from the special `gh-pages` branch of the repo, which contains only the output of MkDocs. MkDocs has a built-in facility for updating this branch after building the docs. One option may be to add the zip file to a subdir of `docs/src` (maybe `docs/src/downloads/zephyr`). It's no ideal that it would live under `docs`, because it's not a doc, but it's still an Ok option I think. I've pushed PR #276 as a way to do that if you think that works.

**Comment by mogenson** [comment]
Sure that works for me. I didn't know if you wanted that binary committed as part of the git repo or not.
